### PR TITLE
Drawables Size Optimized

### PR DESCRIPTION
**Pull request by codingblazer**
#262 (Part-2)
All non sharp drawables has been reduced in size using tinypng  and  sharp drawables has been replaced by vector drawables  wherever possible.

Raw file Size **Decreased by 2 MB** (from 3.7 MB to 1.7 MB) for Drawables in Apk Size Distribution.

### CODE REVIEW

File: source-code/app/src/main/res/drawable/ic_assignment_black_24dp.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (opticod) [inline source-code/app/src/main/res/drawable/ic_assignment_black_24dp.xml]: Looking really big in simulator.

File: source-code/app/src/main/res/drawable/ic_assignment_black_24dp.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:height="24dp" android:viewportHeight="24.0" android:viewportWidth="24.0" android:width="24dp">
```
Comment (opticod) [inline source-code/app/src/main/res/drawable/ic_assignment_black_24dp.xml]: Looking really big in simulator.

File: source-code/app/src/main/res/drawable/ic_group_work_black_24dp.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (opticod) [inline source-code/app/src/main/res/drawable/ic_group_work_black_24dp.xml]: Shade is different

File: source-code/app/src/main/res/drawable/ic_input_white_48dp.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (opticod) [inline source-code/app/src/main/res/drawable/ic_input_white_48dp.xml]: shade is again not perfect. Looks like more brighter.

File: source-code/app/src/main/res/drawable/ic_keyboard_capslock_black_24dp.xml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (opticod) [inline source-code/app/src/main/res/drawable/ic_keyboard_capslock_black_24dp.xml]: Again shade.

File: source-code/app/src/main/res/drawable/ic_group_work_black_24dp.xml (hunk 3 above)
Comment (codingblazer) [inline source-code/app/src/main/res/drawable/ic_group_work_black_24dp.xml]: Yes doing it

File: source-code/app/src/main/res/drawable/ic_keyboard_capslock_black_24dp.xml (hunk 5 above)
Comment (codingblazer) [inline source-code/app/src/main/res/drawable/ic_keyboard_capslock_black_24dp.xml]: No, its same as previous one. I have checked.

**Comment by codingblazer** [comment]
@opticod I have checked each and these need modification - library, group, assignment, input. I am changing it, please review.

File: source-code/app/src/main/res/drawable/ic_record_voice_over_black_48dp.xml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (opticod) [inline source-code/app/src/main/res/drawable/ic_record_voice_over_black_48dp.xml]: Revert back for this.

File: source-code/app/src/main/res/drawable/ic_keyboard_capslock_black_24dp.xml (hunk 5 above)
Comment (opticod) [inline source-code/app/src/main/res/drawable/ic_keyboard_capslock_black_24dp.xml]: Revert back for this.

File: source-code/app/src/main/res/drawable/ic_input_white_48dp.xml (hunk 4 above)
Comment (opticod) [inline source-code/app/src/main/res/drawable/ic_input_white_48dp.xml]: Revert back for this.

**Comment by opticod** [comment]
@codingblazer , Mistake, revert back for 3
1. Assignment (all dpi)
2. Input (all dpi)
3. Record_voice(all dpi)

And can you add vector for ```ic_keyboard_capslock_black_down_24dp.png``` as u have did for ```ic_keyboard_capslock_black_24dp.png```?

**Comment by opticod** [comment]
@codingblazer, Lets fix this, and freeze a version for testing.

**Comment by codingblazer** [comment]
@opticod I am doing this only right now. Will take max 5-10 min.

**Comment by codingblazer** [comment]
@opticod Done. Also added the Caps_down you asked for.

**Pull request merged**